From 219e7ed8f795730906fb04b588b8174e73c7bb57 Mon Sep 17 00:00:00 2001
From: Matt Roper <matthew.d.roper@intel.com>
Date: Wed, 14 Dec 2016 14:51:31 -0800
Subject: [PATCH 3931/4706] drm: Ensure drm_clip_rect is defined

commit e946bd2c504a29543bd149b850c6c67f4047b445 from
git://git.yoctoproject.org/linux-yocto-4.1

drm_framebuffer.h references this structure in parameters, but doesn't
have a #include for any headers that define it.  We wind up relying on a
.c file to already have included the necessary headers.

This isn't necessary on the upstream kernel, but due to history
flattening on our forklift, there are a few places where we lack the
proper include chain, so adding the explicit forward declaration fixes
this easily.

Signed-off-by: Matt Roper <matthew.d.roper@intel.com>
---
 include/drm/drm_framebuffer.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/drm/drm_framebuffer.h b/include/drm/drm_framebuffer.h
index 50deb40..1fecfd2 100644
--- a/include/drm/drm_framebuffer.h
+++ b/include/drm/drm_framebuffer.h
@@ -30,6 +30,7 @@
 struct drm_framebuffer;
 struct drm_file;
 struct drm_device;
+struct drm_clip_rect;
 
 /**
  * struct drm_framebuffer_funcs - framebuffer hooks
-- 
1.7.5.4

